satellite/satellitedb/overlaycache.go

```diff
@@ -41,17 +41,15 @@ func (cache *overlaycache) SelectStorageNodes(ctx context.Context, count int, cr
 	nodeType := int(pb.NodeType_STORAGE)
 
 	safeQuery := `
-		WHERE type = ? AND free_bandwidth >= ? AND free_disk >= ?
+		WHERE type = ? AND NOT disqualified
```

Comment:
Because the rest of the columns of the WHERE clause are one per line, I would also break these 2 one per line and probably leave in the first line the `NOT disqualified`